sa/database.go

```diff
@@ -85,59 +85,6 @@ func InitWrappedDb(config cmd.DBConfig, scope prometheus.Registerer, logger blog
 	return dbMap, nil
 }
 
-// InitSqlDb constructs a *sql.DB object using the provided settings and enables
-// 'interpolateParams' and 'parseTime'. If scope is non-Nil database metrics
-// will also be initialized. The only required parameter is config.
-func InitSqlDb(config cmd.DBConfig, scope prometheus.Registerer) (*sql.DB, error) {
-	url, err := config.URL()
-	if err != nil {
-		return nil, fmt.Errorf("failed to load DBConnect URL: %s", err)
-	}
-
-	conf, err := mysql.ParseDSN(url)
-	if err != nil {
-		return nil, fmt.Errorf("while parsing DSN from 'DBConnectFile': %s", err)
-	}
-
-	if len(conf.Params) == 0 {
-		conf.Params = map[string]string{
-			"interpolateParams": "true",
-			"parseTime":         "true",
-		}
-	} else {
-		conf.Params["interpolateParams"] = "true"
-		conf.Params["parseTime"] = "true"
-	}
```

Comment:
Clearly InitSqlDb isn't getting called anywhere, so great to delete this.

But I was surprised to find that its counterpart, InitWrappedDb, doesn't set `interpolateParams=true` or `parseTime=true`. Both default to false.

Things would still work with `interpolateParams=false`, but wouldn't `parseTime=false` cause problems with any of our tables that have a DATE or DATETIME? Clearly not, since our integration tests are passing and have been for as long as this code path has been dead.

https://github.com/go-sql-driver/mysql

> parseTime=true changes the output type of DATE and DATETIME values to time.Time instead of []byte / string The date or datetime like 0000-00-00 00:00:00 is converted into zero value of time.Time.

Maybe this is only relevant when outputting into an `interface{}` value or field rather than a typed `*time.Time` field? I am confused.

Reply:
It does get set, just in a different way: `InitWrappedDb` calls `NewDbMap` which calls `NewDbMapFromConfig` which calls `adjustMySqlConfig` which sets `conf.ParseTime = true`: https://github.com/letsencrypt/boulder/blob/86c8a23a1a6336cacade8193dbc1b953cf66d905/sa/database.go#L219

Reply:
aha, that explains why my case-sensitive search for `parseTime` wasn't finding it!